pkg/checks/traceroute/check.go

```diff
@@ -49,14 +50,16 @@ type tracerouteConfig struct {
 	MaxHops int
 	Rc      helper.RetryConfig
 }
-type tracerouteFactory func(ctx context.Context, cfg tracerouteConfig) (map[int][]Hop, error)
 
-type result struct {
-	// The minimum number of hops required to reach the target
-	MinHops int `json:"min_hops" yaml:"min_hops" mapstructure:"min_hops"`
-	// The path taken to the destination
-	Hops map[int][]Hop `json:"hops" yaml:"hops" mapstructure:"hops"`
-}
+type (
+	tracerouteFactory func(ctx context.Context, cfg tracerouteConfig) (map[int][]Hop, error)
+	result            struct {
+		// The minimum number of hops required to reach the target
+		MinHops int `json:"min_hops" yaml:"min_hops" mapstructure:"min_hops"`
+		// The path taken to the destination
+		Hops map[int][]Hop `json:"hops" yaml:"hops" mapstructure:"hops"`
+	}
+)
```

Comment:
Imo this syntax is rather ugly for declaring multiple types but I care too little about it to be annoyed from it, so keep it this way if you like it better.

Reply:
golangci lint takes care of formatting. I'm just a mindless drone. ;)